src/main/groovy/io/ebean/gradle/EbeanEnhancer.groovy

```diff
@@ -54,6 +52,7 @@ class EbeanEnhancer {
         classInputStream.close()
 
         String jvmClassName = className.replace('.','/')
+        logger.info("Try to enhance $jvmClassName");
```

Comment:
Do we really want this at info level?  I might drop this down to debug afterwards.